lib/segment/src/index/field_index/full_text_index/mmap_inverted_index/mmap_postings.rs

```diff
@@ -83,45 +176,56 @@ impl MmapPostings {
             .map(|(header, _)| header)
     }
 
-    /// Create ChunkReader from the given header
+    /// Create PostingListView<V> from the given header
     ///
     /// Assume the following layout:
     ///
     /// ```ignore
     /// last_doc_id: &'a PointOffsetType,
     /// chunks_index: &'a [PostingChunk<()>],
     /// data: &'a [u8],
+    /// var_size_data: &'a [u8], // might be empty in case of only ids
     /// _alignment: &'a [u8], // 0-3 extra bytes to align the data
     /// remainder_postings: &'a [PointOffsetType],
     /// ```
     fn get_view<'a>(
         &'a self,
-        header: &PostingListHeader,
+        header: &'a PostingListHeader,
         hw_counter: ConditionedCounter<'a>,
-    ) -> Option<IdsPostingListView<'a>> {
+    ) -> Option<PostingListView<'a, V>> {
         let counter = hw_counter.payload_index_io_read_counter();
 
         let bytes = self.mmap.get(header.offset as usize..)?;
         counter.incr_delta(size_of::<PointOffsetType>());
 
         let (last_doc_id, bytes) = PointOffsetType::read_from_prefix(bytes).ok()?;
 
-        counter.incr_delta(size_of::<PostingChunk<()>>());
+        counter.incr_delta(size_of::<PostingChunk<<V::Handler as ValueHandler>::Sized>>());
         let (chunks, bytes) =
-            <[PostingChunk<()>]>::ref_from_prefix_with_elems(bytes, header.chunks_count as usize)
-                .ok()?;
-        let (data, bytes) = bytes.split_at(header.data_bytes_count as usize);
+            <[PostingChunk<<V::Handler as ValueHandler>::Sized>]>::ref_from_prefix_with_elems(
```

Comment:
If possible, let's try to use type alias for `V::Handler as ValueHandler>::Sized>` as it becomes bulky. Also, it would be nic eto mention in comments next to the alias definition an example of typical type which is actually going to be used there, as it simplifies intuition